tests/models/test_graph.py

```diff
@@ -0,0 +1,85 @@
+import tempfile
+import shutil
+import os
+
+import h5py
+from pdb2sql import pdb2sql
+import numpy
+
+from deeprank_gnn.models.grid import GridSettings, MapMethod
+from deeprank_gnn.models.graph import Graph, Edge, Node
+from deeprank_gnn.models.contact import ResidueContact
+from deeprank_gnn.tools.pdb import get_structure
+from deeprank_gnn.domain.amino_acid import *
+from deeprank_gnn.tools.grid import HDF5KEY_GRID_MAPPEDFEATURES, HDF5KEY_GRID_MAPPEDFEATURESVALUE
+from deeprank_gnn.tools.graph import HDF5KEY_GRAPH_NODEFEATURES, HDF5KEY_GRAPH_EDGEINDICES, HDF5KEY_GRAPH_EDGEFEATURES
+
+
+def test_graph_build_and_export():
```

Comment:
Nice test! Since it is quite long, you could consider adding a docstring describing the different steps.

Reply:
docstring added